composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php":              ">=5.3.2",
-        "symfony/console":   "~2.1"
+        "symfony/console":   ">=2.1"
```

Comment:
@TiGR can you use this form instead: `~2.3|~3.0`? I think it's ok to use `2.3` instead of `2.1` too (though I don't really care). If you make this change, I can merge and make a release :)